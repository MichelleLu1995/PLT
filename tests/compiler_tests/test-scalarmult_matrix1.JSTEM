def int main() {
	float[3][3] m1;
	float[3][3] m2;

	int i;
	int j;

	m1 = {% 1.0, 2.0, 3.0 | 2.1, 2.2, 2.3 | 3.1, 3.2, 3.3 %};

	m2 = m1 * 2.2;
	print_matrixf($$m2, 3, 3);

	return 0;
}
