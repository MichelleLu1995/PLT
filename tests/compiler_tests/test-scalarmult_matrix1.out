{ 2.200000, 4.400000, 6.600000
  4.620000, 4.840000, 5.060000
  6.820000, 7.040000, 7.260000 }
